internal/key/key.go

```diff
@@ -75,19 +75,19 @@ func ReadConfigMapYamlFromFile(fs afero.Fs, path string) (string, error) {
 }
 
 // readSecretFromFile reads a configmap from a YAML file.
-func ReadSecretYamlFromFile(fs afero.Fs, path string) (map[string][]byte, error) {
+func ReadSecretYamlFromFile(fs afero.Fs, path string) ([]byte, error) {
 	data, err := afero.ReadFile(fs, path)
 	if err != nil {
 		return nil, microerror.Mask(err)
 	}
 
-	rawMap := map[string][]byte{}
+	rawMap := map[string]interface{}{}
 	err = yaml.Unmarshal(data, &rawMap)
```

Comment:
Is it just validating purpose? then what do you think to use an anonymous variable?
```suggestion
	err = yaml.Unmarshal(data, &map[string]interface{}{})
```

Reply:
Good catch and yes we only need this for validating. Thx!